src/pages/home/sidebar/SidebarLinks.js

```diff
@@ -189,6 +193,7 @@ class SidebarLinks extends React.Component {
                         shouldDisableFocusOptions={this.props.isSmallScreenWidth}
                         optionMode={this.props.priorityMode === CONST.PRIORITY_MODE.GSD ? 'compact' : 'default'}
                         onLayout={() => {
+                            this.props.onLayout();
```

Comment:
not sure why it will fail unit testing? I set for `onLayout` a default prop, so that shouldn't be an issue?  
(Also when running the tests manually they all succeed 👀 )